### PR TITLE
feature/OE-201-manager-listing-users-index

### PR DESCRIPTION
**Pull request by devsquad-mathews-lima**
## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: resources/views/livewire/castle/users.blade.php [hunk 1]
```diff
@@ -31,10 +31,10 @@
                                             @lang('Email')
                                         </x-table.th>
                                         <x-table.th-searchable by="role" :sortedBy="$sortBy" :direction="$sortDirection">
-                                            @lang('Role')
+                                            @lang('Name')
                                         </x-table.th>
```
Comment (Ron-Junior) [inline resources/views/livewire/castle/users.blade.php:35]: the table will have two columns with "Name"?

**Review by Ron-Junior — CHANGES_REQUESTED** [review]
